canvas_modules/common-canvas/src/toolbar/toolbar-overflow-item.jsx

```diff
@@ -53,6 +53,7 @@ class ToolbarOverflowItem extends React.Component {
 						onClick={this.toggleExtendedMenu}
 						onFocus={this.props.onFocus}
 						aria-label={this.props.label}
+						size={this.props.size}
```

Comment:
You need to set `hasIconOnly` in the props here. You shouldnt need to have so many style overrides just for this button.